foreman.fc

```diff
@@ -41,6 +41,8 @@
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
```

Comment:
For a separate PR: we can clean up the 1.8 ruby contexts since that's EL6.

Reply:
We could, however this is such a mess that I suggest to wait until we get rid of Passenger which is really happening now and then delete all of it :-)